cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -40,24 +39,16 @@ func main() {
 }
 func run() error {
 	// TODO(lingshi) Move this to a shared place across Linux and Windows.
-	builtInConfig, mergedConfig, err := confgenerator.MergeConfFiles(*input, "linux", apps.BuiltInConfStructs)
+	uc, err := confgenerator.MergeConfFiles(*input, "linux", apps.BuiltInConfStructs)
 	if err != nil {
 		return err
 	}
 
 	// Log the built-in and merged config files to STDOUT. These are then written
 	// by journald to var/log/syslog and so to Cloud Logging once the ops-agent is
 	// running.
-	log.Printf("Built-in config:\n%s", builtInConfig)
-	log.Printf("Merged config:\n%s", mergedConfig)
+	log.Printf("Built-in config:\n%s\n", apps.BuiltInConfStructs["linux"])
```

Comment:
Sorry, I was wrong; `fmt.Printf` doesn't add the new lines but the `log` package actually prints everything on a new line regardless. Sorry!